worlds/yoshisisland/Items.py

```diff
@@ -0,0 +1,125 @@
+from typing import Dict, Set, Tuple, NamedTuple
+
+class ItemData(NamedTuple):
+    category: str
+    code: int
+    count: int = 1

```

Comment:
`count` is the name of a method on `tuple`
Since this inherits from tuple, this is an invalid override of `count`

You might want a different name for this field.